pmd/defs.bzl

```diff
@@ -11,15 +11,15 @@ def _impl(ctx):
     # Sources
 
     if len(ctx.files.srcs) != 0:
-        srcs_file = _write_files_list(ctx, ctx.files.srcs, "srcs.txt")
+        srcs_file = _write_files_list(ctx, ctx.files.srcs, "srcs_%s.txt" % ctx.label.name)
```

Comment:
Can I ask you to use `"srcs_{}".format(ctx.label.name)` here and below? There is no clear benefit, just a style choice.

Reply:
I've updated the formatting to match the style you've used throughout the rest of the project.

I'm finding the tests are not passing. It's a combination of things. 
* The `"srcs_{}".format(ctx.label.name)` doesn't work well in the tests. The format command is compromised by the `{{output_dir}}` and `{{source_dir}}` processing that occurs.
* The other issue is that the `ctx.label.name` is not matching the expected name. The names are getting switched around a bit.

https://github.com/jmeekhof/bazel_rules_pmd/blob/srcs_naming/tests/analysis/tests.bzl#L191-L201

This shows one of the failing pieces.

Reply:
ok -- I figured out the testing. The name attribute in the test needs to come from the name of the rule under test, not the name of the rule _doing_ the test.